webSocket/transceiver/transceiver.go

```diff
@@ -103,7 +112,12 @@ func (wt *wsTransceiver) Listen(connection webSocket.WSConClient) (closed bool)
 	}
 }
 
-func (wt *wsTransceiver) verifyPayloadAndSendAckIfNeeded(connection webSocket.WSConClient, payload []byte) {
+func (wt *wsTransceiver) verifyPayloadAndSendAckIfNeeded(connection webSocket.WSConClient, payload []byte, msgType int) {
+	if msgType == websocket.TextMessage {
```

Comment:
so you rely on binary messages for payloads, and text messages for acks? Interesting idea, but I think the old way was more extendable

Reply:
refactored this part.